contrib/docs/evm-v0.3-upgrade.md

```
@@ -0,0 +1,145 @@
+
+It's important to highlight we went with a different approach for the chain Id changes, mimicking mantrachain. This required some back and forth fixes we were not expecting. 
+We are also using our own fork of cosmos/evm  v0.4.1 with changes to fee collection to allow feeless transactions.
+
+## Requirements
+- IBC v10
+### Import changes
+- Update vm core usage:
+  - "github.com/cosmos/evm/x/vm/core/vm" -> "github.com/ethereum/go-ethereum/core/vm"
+- Replace go ethereum with fork in go.mod:
+```go
+	// use Cosmos geth fork
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)